### ISSUE TITLE
When is a description a prediction and when is it an observation?

### ISSUE THREAD

**Issue by ifokkema** [HGVSnomenclature/hgvs-nomenclature#146]
The documentation states in several places that RNA descriptions should be surrounded by parentheses if "RNA has not been analysed". However, it is not specified how RNA should be analyzed or in what context. This leaves room for interpretation. Within the LOVD team and between submitters, this rule is interpreted in several ways. For example, a mini gene assay would be sufficient to provide evidence for an _in vitro_ database record, but not everybody agrees whether this assay provides evidence for the RNA change in an actual patient.

After discussion in the HVNC meeting on February 12th, 2024, it was decided that it was not the place for the HVNC to define when an experiment provided sufficient evidence for the parentheses to be removed. As such, it will be left to the implementor of the variant description to make this consideration. We should add this to the documentation to highlight this problem so that people are aware of it.

### Suggested implementation:
- [ ] Find a good place to explain this situation (suggestion: `recommendations/general.md`).
- [ ] Add text, e.g., "NOTE: the HVNC does not provide rules or regulations when an experiment provided sufficient evidence for a description to be considered not a prediction, but an observed change. This is left to the implementor of the variant description."
- [ ] Add a `<a id="..."></a>` tag so we can link to this text.
- [ ] Make sure all relevant pages mention the rule about predictions requiring parentheses (not al pages contain this info).
    - [ ] Add this to the DNA pages as well?
- [ ] Add a link to the detailed explanation, e.g., "(see notes in the general recommendations)".
- [ ] This may also solve the discussion of whether observing a change on RNA level is sufficient to consider a description on protein level an observation and not a prediction. We may choose to add this to the general recommendations page as well.

**Comment by jtdendunnen** [comment]
I do not agree with item 2 "NOTE: the HVNC does not provide rules or regulations ..". In my opinion, an in vitro mini-gene assay to study the consequences of a variant on RNA level is not enough to remove the parentheses in an r. description.  Parentheses can only be removed when RNA isolated from the sample of the individual which has the variant is analysed, i.e. RNA is directly sequenced or cDNA (a DNA copy of the RNA) is sequenced.

NOTE: LOVD has specific "in vitro analysis" records, in such a record the parenthesis may be removed).

**Comment by ifokkema** [comment]
> I do not agree with item 2 "NOTE: the HVNC does not provide rules or regulations ..". In my opinion, (...)

As no definitions are currently stated on the website, the actual contents of the HGVS nomenclature website doesn't state any requirements for what "RNA analysed" means. We once discussed this in February, and then the decision was that we wouldn't state any specific rules. We would otherwise have to define this for every molecule. I do not remember whether we voted on this subject or just agreed on that call, which means that if you disagree, we may need to repeat the discussion to get an official decision.
